src/content/docs/dropins/all/enriching.mdx

```diff
@@ -1,6 +1,169 @@
+
+### Commerce blocks
+
+Commerce blocks are Edge Delivery Service blocks that contain drop-in components. Commerce blocks are used to add drop-in components like the PDP drop-in, as well as custom components like product recommendations and other commerce-related information on a webpage.
+
+
+### Enrichment blocks
+
+Enrichment blocks are specialized content blocks to dynamically display content above or below commerce blocks based on product SKUs and category numbers.
+
+### Content positioning
+
+Content positioning is the placement of content and enrichment blocks on a webpage. It includes the arrangement of content blocks above or below a commerce blocks. Positioning content inside a drop-in component is also possible using the slot API functions. The webpage structure is within a Word or Google docs file. This file includes tables that define content blocks, commerce blocks, and other HTML elements.
+
+### Slot API functions
+
+The slots API provides several functions to add content within a drop-in component by specifying a slot name and a position. The slot API provides functions to add content to specific positions above and below the slot, such as prepend and append. See the full list of slot API functions here: [Extending drop-in components](/dropins/all/extending#big-picture)
+
+</Vocabulary>
+
+## Add content blocks
+
+The following steps show you how to enrich a commerce block by placing content blocks above or below it. The following example shows the default product-details document with a simple product recommendation block below it.
+
+<Tasks>
+
+<Task>
+### Open a commerce block document
+
+Open the document that contains the commerce block you want to enrich. In the boilerplate starter content, open the `products/default.docx` to see an example of a product recommendation block below the product details block.
+
+</Task>
+
+<Task>
+### Add normal content or content blocks
+
+Add a normal headings, text, images or content blocks above or below the commerce block in the document. You can copy content blocks (tables) from the [AEM Sidekick Library](https://sidekick-library--aem-block-collection--adobe.aem.page/tools/sidekick/library.html?plugin=blocks&path=/block-collection/) and paste them into the document.
+
+</Task>
+
+<Task>
+### Preview and publish changes
+
+Use AEM Sidekick to preview and publish the changes to see the content above or below the commerce block.
+
+</Task>
+
+</Tasks>
+
+## Add enrichment blocks
+
+Enrichment blocks are used to add content above or below commerce blocks based on product and category numbers. The following steps show you how to enrich a commerce block by placing an enrichment block above or below it.
+
+<Tasks>
+
+<Task>
+### Create enrichment content
+
+In Google Docs or SharePoint, add a new document to the `enrichment` folder. You can duplicate the `product-banner` document and rename it. Then add the content you want to display above or below the commerce block.
```

Comment:
Guidance needed. Give an example of what I might do.

Reply:
Yeah, I'll add a banner of some sort.